snippets/core/whats-new/whats-new-in-30/cs/Program.cs

```diff
@@ -102,6 +102,9 @@ public static void ReadJson(string jsonString)
                 var name = config.GetProperty("name").GetString();
                 Console.WriteLine($"Config: {name}");
             }
+
+            JsonPerson[] a;
+            a = { { new JsonPerson() } };
```

Comment:
@Thraka that addition looks strange: declaration and initialization are separated; also `a` is not used in the snippet.

Reply:
Oh weird. I think this file snuck in this PR, it doesn't belong

Reply:
Thanks @pkulikov for finding that 😄